src/frost.rs

```diff
@@ -437,9 +437,12 @@ impl SignatureShare {
 /// perform the first round. Batching entails generating more than one
 /// nonce/commitment pair at a time.  Nonces should be stored in secret storage
 /// for later use, whereas the commitments are published.
+
+/// The number of nonces is limited to 255. This limit can be increased if it
+/// turns out to be too conservative.
```

Comment:
This is no longer enforced by the type (u8) so we should either change this doc comment or enforce it in the code

Reply:
The  `num_nonces` is still `u8`: https://github.com/ZcashFoundation/redjubjub/blob/21b2f0863dbf2af72ec242f2d033f856143b7336/src/frost.rs#L444
I think the comment is still valid.

Reply:
What was once enforced by the type system is not anymore in this PR.

Reply:
Sorry, it is still not clear to me. Can you make a suggestion on how the comment should be ? I am thinking on just removing it.

Reply:
Let's:
- create an issue to 'enforce maximum of 256 participants with u64 indexes'
- mention that as a TODO in this comment

Then I think that'll be fine to get this change merged, that ticket should also be resolved in Marek's PR

Reply:
Sounds good, i created the ticket and added a suggestion here to change the comment.

Reply:
Note that the number of these nonces is not really related to the number of participants. These nonces are generated for each participant during the preprocessing stage. Our "message protocol" currently supports only one pair of nonces per signature so that the nonces (and corresponding commitments) are not cached and signing is always two-round.
(Here's the rationale for omitting cached nonces for now: https://github.com/ZcashFoundation/redjubjub/pull/85#discussion_r631458009)